File: 3345-smallest-divisible-digit-product-i/3345-smallest-divisible-digit-product-i.py
class Solution:
    def smallestNumber(self, n: int, t: int) -> int:
        temp=0
        for i in range(n,101):
            num=1
            temp2=i
            while temp2>0:
                rem=temp2%10
                num*=rem
                temp2//=10
            if(num%t==0):
                temp=i
                break
        return temp

File: 3345-smallest-divisible-digit-product-i/3345-smallest-divisible-digit-product-i.java
class Solution {
    public int smallestNumber(int n, int t) {
        int temp=0;
        for(int i=n;i<=100;i++){
            int num=1;
            int temp2=i;
            while(temp2>0){
                int rem=temp2%10;
                num*=rem;
                temp2/=10;
            }
            if(num%t==0){
                temp=i;
                break;
            }
        }
        return temp;
    }
}